TODO

```diff
@@ -1,9 +1,10 @@
-
 https://github.com/ericmj/postgrex/blob/5037f45214cff04738ee3d359832443dfaa1494a/test/query_test.exs#L529
-=> stream a reopened statement is ok
-
+=> "stream a reopened statement is ok"
 
 https://github.com/ericmj/postgrex/blob/5037f45214cff04738ee3d359832443dfaa1494a/test/query_test.exs#L539
+=> can re execute a closed query
```

Comment:
We want to prepare, close the query and then try to run the stream. This should work as we will need to automatically (re)prepare the query on the initial execute.